### PR TITLE
Fix encoding issues

### PR DESCRIPTION
**Pull request by labkey-tchad**

### CODE REVIEW

File: assay/src/org/labkey/assay/view/newUploadBatchProperties.jsp [hunk 1]
```diff
@@ -36,7 +36,7 @@
 %>
     <labkey:input type="displayfield"
           label="<%= entry.getKey().getPropertyDescriptor().getNonBlankCaption() %>"
```
Comment (labkey-jeckels) [inline assay/src/org/labkey/assay/view/newUploadBatchProperties.jsp:38]: @labkey-tchad @labkey-adam should the label be encoded too?

**Review by labkey-jeckels — CHANGES_REQUESTED** [review]

File: assay/src/org/labkey/assay/view/newUploadBatchProperties.jsp (hunk 1 above)
Comment (labkey-tchad) [reply assay/src/org/labkey/assay/view/newUploadBatchProperties.jsp:38]: LabKey taglibs auto-encode most attributes, just not `value`.

Comment (labkey-jeckels) [reply assay/src/org/labkey/assay/view/newUploadBatchProperties.jsp:38]: Ugh!

**Review by labkey-jeckels — APPROVED** [review]
I know we're waiting for another round of results, but what's here looks fine.

**Pull request merged**